artiq/coredevice/mirny.py

```diff
@@ -78,15 +98,16 @@ def init(self, blind=False):
 
         :param blind: Do not attempt to verify presence and compatibility.
```

Comment:
The effect of `blind` on the clk_sel choice should be documented.

Reply:
Is there an effect of ``blind`` on the ``clk_sel`` choice? There shouldn't.

Reply:
Ah. I see that you have taken the read out of the blind branch. All good.